lib/const.rb

```diff
@@ -284,7 +280,7 @@
   -DCMAKE_C_FLAGS='#{CREW_COMMON_FLAGS.gsub(/-fuse-ld=.{2,4}\s/, '')}' \
   -DCMAKE_CXX_FLAGS='#{CREW_COMMON_FLAGS.gsub(/-fuse-ld=.{2,4}\s/, '')}' \
   -DCMAKE_EXE_LINKER_FLAGS='#{CREW_LINKER_FLAGS}' \
-  -DCMAKE_LINKER_TYPE=#{CREW_LINKER.upcase} \
+  -DCMAKE_LINKER_TYPE=MOLD \
```

Comment:
Not variable?

Reply:
I have removed it due to the lack of use cases. Why use other linkers if there is a much faster one :)

Reply:
Reverted.